### ISSUE TITLE
NFR: Language setting needs to be applied to all website - MT

### ISSUE THREAD

**Issue by ZoeBeykirch** [aemsites/vg-macktrucks-com#100]
### Description 
As a Mack website owner, I want to ensure that all my websites have language settings so that we can improve accessibility and meet WCAG level A requirements. 

### Acceptance Criteria 
- [ ] The <html> tag in the HTML code of all websites must include the correct lang attribute, either lang="en" or lang="en-us", depending on the language and region.
- [ ] The language setting must be added consistently across all websites and market sites for Mack. 
- [ ] The change must pass a quick accessibility check via CrownPeak's accessibility crawler (https://dqm.crownpeak.com/group/accessibility).

### Notes
(From Syb) Hi Team, Seems like all pages for VTNA & Mack has no language setting, " < html lang = "en" > " or " < html lang=" en-us ">", this affects accessibility (WCAG level A). Probably this is a generic template thing and thus easy to fix.

I did follow up on CrownPeak utilization - and happened to observe...
Pages are crawled by CrownPeak "https://dqm.crownpeak.com/group/accessibility", so anyone with access can do a quick check, please tell me if I should add users in CrownPeak for any walkthrough of all issues.

**Comment by Meli-lazarte** [comment]
Discuss this further with @ZoeBeykirch.

**Comment by ZoeBeykirch** [comment]
@Meli-lazarte Yes this is an old one but should still need to be addressed. I just updated the ticket to have a better description and acceptance criteria.. it's for all of the brands so we may need to add dedicated tickets for volvo and road choice?

**Comment by Meli-lazarte** [comment]
Ticket created for Volvo Trucks https://github.com/aemsites/volvotrucks-na/issues/169 and for Road Choice https://github.com/aemsites/vg-roadchoice-com/issues/114.

**Comment by ZoeBeykirch** [comment]
Thank you!!

**Comment by ZoeBeykirch** [comment]
looks like the report is not considering "en-US" still shows 112 issues.. see attached. 
[Mack_Trucks_NA_(EN-US)_Issue-Summary_03-Jan-2025.csv](https://github.com/user-attachments/files/18301291/Mack_Trucks_NA_.EN-US._Issue-Summary_03-Jan-2025.csv)

Kannan is checking with the central team on what is considered correct lang="en" of lang"en-US"

**Comment by Meli-lazarte** [comment]
@Kannan-CM any update on this?

**Comment by Kannan-CM** [comment]
Hi @Meli-lazarte ,

Using "en-US" is also an accepted format.
I digged into to the issue further and found and interesting difference.

The HTML doesn't have the attribute with the page source is viewed. Please see the below screenshot highlighting the difference.

<img width="949" alt="Image" src="https://github.com/user-attachments/assets/7f73a791-fc84-43bd-ae6e-31cda3c32200" />

**Comment by silviulungu** [comment]
The testing is completed and everything works accordingly.
Just one content issue i would like to mention and that is on french Canada the mega menu is in english.

<img width="1722" alt="Image" src="https://github.com/user-attachments/assets/2d14eca2-36bb-4879-8b1b-7aad0f1c43a7" />
